### PR TITLE
Fix job name for fetch jobs

### PR DESCRIPTION
**Pull request by dv-blk**
#### Purpose
My nodejs command now has a memory argument.
`/usr/local/bin/node --max-old-space-size=8192 /app/src/runners rawMetrics`

This causes the name to become `/app/src/runners`.

I'm changing this script to set the last word to the name rather than word 2.

#### Notes
Left some bash explainers, cleaned up the code a bit, tested locally.

### CODE REVIEW

File: deploy_to_kubernetes_cron.sh [hunk 1]
```diff
@@ -1,38 +1,40 @@
...
-JOB_TEMPLATE="/deploy/kubernetes/cron.job.template.yaml"
-JOB_FILE="/deploy/kubernetes/cron.job.yaml"
 
+# NAME, commands
 source /deploy/kubernetes/commands.sh && get_commands
 source /deploy/kubernetes_deploy_base.sh
 
-JOB_NAME=$NAME
+JOB_TEMPLATE="/deploy/kubernetes/cron.job.template.yaml"
+TEMPORARY_JOB_FILE="/deploy/kubernetes/cron.job.yaml"
```
Comment (jakeorr) [inline deploy_to_kubernetes_cron.sh]: can this include "temp" in the name?

File: deploy_to_kubernetes_cron.sh [hunk 2]
```diff
@@ -1,38 +1,40 @@
...
 
-export PRIMARY_IMAGE="${PRIMARY_IMAGE}"
+# `commands` are an associative array -- [COMMAND]=SCHEDULE
 
+# Get all keys, each as a new string: "${!commands[@]}" 
 for command in "${!commands[@]}"; do
+
+  # Split string into array of strings
   split_command=($command)
-  formatted_command=\"${split_command[0]}\"
```
Comment (jakeorr) [inline deploy_to_kubernetes_cron.sh:16]: Are we losing some functionality here?

File: deploy_to_kubernetes_cron.sh (hunk 1 above)
Comment (dv-blk) [reply deploy_to_kubernetes_cron.sh]: You mean to redact `ORARY`?

File: deploy_to_kubernetes_cron.sh (hunk 2 above)
Comment (dv-blk) [reply deploy_to_kubernetes_cron.sh:16]: Oh right, this needs to be csv. Update in a moment.

File: deploy_to_kubernetes_cron.sh (hunk 1 above)
Comment (jakeorr) [reply deploy_to_kubernetes_cron.sh]: I meant in the filename.

Comment (dv-blk) [reply deploy_to_kubernetes_cron.sh]: Oh yup

**Comment by dv-blk** [comment]
updated

**Review by jakeorr — APPROVED** [review]
Nice. Good comments. 👍

**Pull request merged**